db/seeds.rb

```diff
@@ -218,14 +234,16 @@
 end_time_1 = Time.at(79200).utc.strftime("%H:%M:%S")
 start_time_2 = Time.at(39600).utc.strftime("%H:%M:%S")
 end_time_2 = Time.at(75600).utc.strftime("%H:%M:%S")
-# Seed deals for amazingcoders8mc@gmail.com
 
+# Seed deals for amazingcoders8mc@gmail.com
+activate_date = '20-09-2015'.to_datetime.in_time_zone("Singapore") - 8.hours
```

Comment:
@Woonyong92 why is there a need to do this -8.hours conversion? I dn see this for other datetimes.